library/patient_self_checkin/step5.php

```diff
@@ -0,0 +1,188 @@
+<?php
```

Comment:
All you scripts need a standard author/license/copyright header. See your header in library/patient_self_checkin/index.php for a good example.